### DISCUSSION TITLE
Getting this error "Running target start for project @notesnook/web and 9 tasks it depends on failed"

### DISCUSSION THREAD

**Discussion by dotsvm** [streetwriters/notesnook#2498] [General]
![error](https://user-images.githubusercontent.com/117660893/236635340-9d50bef0-6ae3-41f4-9410-be98127360dc.png)

**Comment by xa4hf8** [comment]
It appears that GitHub is refusing the connection.  They might have a misconfigured firewall, or you might be using a blacklisted proxy address.  Does the issue persist ?
